### PR TITLE
Line spacing fixed

### PR DESCRIPTION
**Pull request by AliSinaYOusofi**
## What does this PR do?
Add resource(s) 

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [ x] Add needed indications (PDF, access notes, under construction).
- [ x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AliSinaYOusofi** [comment]
why do I get errors when making a PR. I have read the guidelines.

File: more/free-programming-playgrounds-pr.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+### وب سایت هایی برای یادگیری برنامه نویس وب جلویی
+
+* [javascript_html_css](#javascript_html_css)
+* [cssGrid](#css_grid)
+* [react](#react)
+* [react-hooks](#react)
+* [Next](#nextjs)
+
+### javascript_html_css
```
Comment (bugbunnyy) [inline more/free-programming-playgrounds-pr.md:9]: There should be two new lines before the heading here you gave only one need to add another and same for other heading as well

Comment (bugbunnyy) [reply more/free-programming-playgrounds-pr.md:9]: Let me know if you still face any issue. Thanks

Comment (AliSinaYOusofi) [reply more/free-programming-playgrounds-pr.md:9]: thanks, it worked.

**Pull request closed without merging**